example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -38,7 +38,7 @@ const { getGranuleWithStatus } = require('@cumulus/integration-tests/Granules');
 const { createCollection } = require('@cumulus/integration-tests/Collections');
 const { createProvider } = require('@cumulus/integration-tests/Providers');
 const { deleteCollection, getCollections } = require('@cumulus/api-client/collections');
-const { deleteGranule } = require('@cumulus/api-client/granules');
+const { deleteGranule, removePublishedGranule } = require('@cumulus/api-client/granules');
```

Comment:
Nit: When bringing in the api-client we should consider sorting moving all of the granulesApiTestUtils calls to use api-client. Scope is a fine reason not to in this PR, however.